webxr-api/device.rs

```diff
@@ -82,6 +84,14 @@ pub trait DeviceAPI<Surface>: 'static {
     }
 
     fn granted_features(&self) -> &[String];
+
+    fn request_hit_test(&mut self, _source: HitTestSource) {
+        panic!("This device does not support requesting hit tests");
```

Comment:
Would a error! be more appropriate? I don't have an opinion on the matter.

Reply:
if this happens it's a bug on our side, it means we either wrote a backend that falsely claimed hit test support, or the script code is not validating features

which reminds me, the mock backend should start reporting that it supports hit testing